### DISCUSSION TITLE
Should `--regex` be True by default?

### DISCUSSION THREAD

**Discussion by rdrey** [godaddy/tartufo#156] [General]
It's weird that we do `--entropy` by default, but not `--regex`. It looks like the regex rules are higher signal to noise than the entropy results (in my experience at least) and it's strange that we default to "fewer results" or the less secure runtime setting.

It also creates a config item we need to set in every `tartufo.toml` or remember to specify at runtime.

**Comment by tarkatronic** [comment]
I think this is a good call. As a matter of fact, I would expand this topic to ask: What options should change with the release of 3.0?

I would vote:

* `--regex` -> `True`
* `--fetch` -> `False`
* ...others?

**Comment by tarkatronic** [comment]
This will be fixed in 3.0 via #254 😄
